gobblin-modules/gobblin-sql/src/main/java/org/apache/gobblin/writer/commands/MySqlWriterCommands.java

```diff
@@ -47,14 +48,17 @@
   private static final String INFORMATION_SCHEMA_SELECT_SQL_PSTMT =
       "SELECT column_name, column_type FROM information_schema.columns WHERE table_schema = ? AND table_name = ?";
   private static final String COPY_INSERT_STATEMENT_FORMAT = "INSERT INTO %s.%s SELECT * FROM %s.%s";
+  private static final String COPY_REPLACE_STATEMENT_FORMAT = "REPLACE INTO %s.%s SELECT * FROM %s.%s";
   private static final String DELETE_STATEMENT_FORMAT = "DELETE FROM %s.%s";
 
   private final JdbcBufferedInserter jdbcBufferedWriter;
   private final Connection conn;
+  private final boolean replaceExistingValues;
 
   public MySqlWriterCommands(State state, Connection conn) {
     this.conn = conn;
     this.jdbcBufferedWriter = new MySqlBufferedInserter(state, conn);
+    this.replaceExistingValues = state.getPropAsBoolean(ConfigurationKeys.REPLACE_FOR_MYSQL_INGESTION);
```

Comment:
I hold a bias toward ctors taking typed params, rather than sifting through config for special keys.  up to you, but I'd read config earlier on the call stack and translate to a `boolean` param, ultimately used here.

Reply:
The place I could provide the boolean param for the ctor looks to be in the constructor for `JdbcWriterCommands`. It does not have much logic other than choosing which class to instantiate. There doesn't seem to be a large benefit from translating the config to a boolean one step earlier so I'll leave as is.